app/forms/flood_risk_engine/steps/null_form.rb

```diff
@@ -16,13 +16,11 @@ def save
           true
         end
 
-        def validate(params)
+        def validate(*)
           true
         end
 
-        def enrollment_id
-          enrollment.id
-        end
+        delegate :id, to: enrollment, prefix: true
```

Comment:
I prefer the original. 

I am a great fan of `delegate` but not in the case, where 

- there is only one method being delegated
- the use of the prefix makes it less obvious that the resulting method will be `enrollment_id`